app/App.tsx

```diff
@@ -33,6 +34,10 @@ export default function App() {
   const theme = scheme === 'dark' ? AppDarkTheme : AppLightTheme
   const [bgColor, setBgColor] = useState("#ffffff")
 
+  useEffect(() => {
+    TrackPlayer.registerService()
+    TrackPlayer.resetPlayer()
+  })
```

Comment:
Tohle by mělo běžet jenom při prvním renderu

```suggestion
  }, [])
```

Reply:
💡